command/do_transition_test.go

```diff
@@ -0,0 +1,98 @@
+package command
+
+import (
+	"encoding/json"
+	"github.com/ExpediaGroup/flyte-jira/client"
+	"github.com/HotelsDotCom/flyte-client/flyte"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+type (
```

Comment:
Valid but not common way to declare structs. This block approach is used mainly for imports sometimes for const.

For types it best to stick to generally use syntax

type MyType struct{}
type MyAnotherType struct{}

Reply:
done